src/include/OpenImageIO/simd.h

```diff
@@ -634,6 +634,34 @@ class int4 {
 #endif
     }
 
+    OIIO_FORCEINLINE void load (const int *values, int n) {
+#if defined(OIIO_SIMD_SSE)
+        switch (n) {
+        case 1:
+            m_vec = _mm_castps_si128 (_mm_load_ss ((const float *)values));
```

Comment:
I've verified on multiple gcc versions that it's ok (though I did find an entirely different warning that I didn't get on clang, and which I have fixed and updated this PR).